src/aboutUs.json

```diff
@@ -0,0 +1,42 @@
+{
+    "sections": [
+      {
+        "title": "Introduction",
+        "content": [
+          {
+            "par": "AnitaB.org Open Source is an international sub-community of AnitaB.org encouraging the participation of all women involved in technology. It has over 2,000 members from more than 15 time zones. We welcome the participation of all genders at any ages and experience levels."
+          }
+        ]
+      },
+      {
+        "title": "History",
+        "content": [
+          {
+            "par": "AnitaB.org Open Source (formerly Systers Open Source) was initiated by Systers, an international community of 6,000 women involved in technical-computing founded by Anita Borg in 1987 as a small electronic mailing list for women in “systems”. We have incredible Systers (Robin Jeffries, Jen Redman, Terri Oda, Leslie Hawthorn, Avni Khatri, and the list goes on!) that paved the way and introduced Systers to open source projects, communities, and technologies. Systers has been involved with open source in various ways, introducing Open Source Day at Grace Hopper Celebration in 2010 at Atlanta, GA (At the time, it was called Codethon for Humanity with only one open source project, Sahana Eden)."
+          },
+          {
+              "par":"Moreover, AnitaB.org Open Source has been participated in other global open source programs such as Google Summer of Code, Google Code In, Outreachy, and more."
+          },
+          {
+            "par": "Today, we have 14 active open source projects, including initiatives with our Peace Corps partners and apps created by Systers members."
+          }
+        ]
+      },
+      {
+        "title": "Mission",
+        "content": [
+          {
+            "par": "AnitaB.org Open Source encourages contributions of ALL kinds (content, design, coding, documentation, testing, etc) from community members. Each member has the opportunity to gain global collaboration experience working on projects with social impact and improve technical skills while participating as a volunteer, mentor, writer, coder, designer, app tester, and more!"
```

Comment:
this content deems fit for meet our open source community rather than our mission 
in the end you can add content like come meet our community and you can find us at zulip github..

Reply:
This is about open source @nandini45 we have portal which says about AnitaB.org. It's good IMHO.

Reply:
but I think our mission is to encourages more girls and women to get in tech.